src/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -80,14 +75,16 @@ class ErrorBoundary extends React.Component<
             ? sendUsageData(error.message)
             : sendGAEvent(error.message);
 
+        const selectedDevice =
```

Comment:
Maybe we should at some point consider changing devices into a `Map` to avoid syntax like this.

Reply:
After reading your comment I realised that this component directly reads the state instead of using selectors here: https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/1f62d64ae751b9f6b78d4450144d44110aaa0ba0/src/ErrorBoundary/ErrorBoundary.tsx#L191-L192

If we switch to selectors, then this component does not really care whether the devices are internally a `Map` or something else. But doing that change proved to be more yak shaving when I realised that the `ErrorBoundary` is used in two different contexts: In apps, where we actually have devices, and in the launcher where there are no devices.

So I postponed doing that update until later and get the PR merged in it's current state.